docs/ENDPOINTS.md

```diff
@@ -57,4 +58,32 @@ To delete a vacancy. **Returns**:
 - 200 OK if the delete was successful
 - 404 Not Found if a product with the specified ID is not found
 
+# Maintenance endpoints (endpoints from maintenance part of Angular front end)
 
+## /getskills
+### GET
+**Returns**
+A list of DTO objects containing the skill names
+
+## /saveskill
+saves a new skill to the skill table.
+### POST
+**Requires**
+a JSON body containing the DTO object of a skill i.e.
+{ 'id' : '',
+  'name':'Angular'
+} 
+the id is not yet used at this point, for our purpose, the id is generated in the back end
+
+## /deleteskill
+skill with the corresponding skill name will be deleted, including the links to vacancies
+### POST
+**Requires** 
+a JSON body containing the DTO object of a skill
+see /saveskill
+
+
+## /relinkskills
```

Comment:
this is not a standard RESTful name, as they should not be verbs, but nouns. 
Take a look at these 3 links:
https://spring.io/guides/tutorials/rest/
https://www.springboottutorial.com/spring-boot-crud-rest-service-with-jpa-hibernate
https://bezkoder.com/spring-boot-jpa-crud-rest-api/

Reply:
I think /SkillMatcher would be good

Reply:
Ok, as long as it does not suggest CRUD operation because it isn't, it is a method to reconcile relationships

Reply:
/skillmatcher because all our endpoints are lowercase only as until now